packages/api/src/templates/password_reset_email.html

```diff
@@ -0,0 +1,146 @@
+<!--
+  Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
+  This file (revocation_of_access_to_farm_email.html) is part of LiteFarm.
+
+  LiteFarm is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  LiteFarm is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ -->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>LiteFarm Farm Access Restored Email</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <style>
+        .background {
+            font-family: "Open Sans", "SansSerif", sans-serif!important;
+            background: #FAFAFD!important;
+            margin: 0!important;
+            padding-bottom: 34px!important;
+        }
+
+        .header {
+            background: #C7EFD3!important;
+        }
+
+        img {
+            display: block!important;
+            margin: auto!important;
+            padding: 0px 28px!important;
+            background: white!important;
+        }
+
+        #logo {
+            height: 48px!important;
+        }
+
+        .main-container {
+            background: white!important;
+            margin: auto!important;
+            width: 95%!important;
+            max-width: 568px!important;
+            border-bottom: 4px solid #C7EFD3!important;
+        }
+
+        .main-content {
+            padding: 2vw 4vw!important;
+        }
+
+        .main-content p {
+            font-style: normal!important;
+            font-weight: normal!important;
+            font-size: 16px!important;
+            line-height: 24px!important;
+            color: #262422!important;
+        }
+
+        .button-aligner {
+            text-align: center!important;
+        }
+
+        .button {
+            -webkit-appearance: button!important;
+            -moz-appearance: button!important;
+
+            text-decoration: none!important;
+            color: #282B36 !important;
+            background: #FCE38D!important;
+            box-shadow: 0px 2px 8px rgba(102, 115, 138, 0.3)!important;
+            border-radius: 4px!important;
+            padding: 12px 100px!important;
```

Comment:
I've made a change to the padding and width of the button, but this has since been reverted. Please restore the change.

Reply:
![Image from iOS](https://user-images.githubusercontent.com/34196752/102528335-404a3380-4053-11eb-9ec4-a78a8d418d7f.jpg)

Reply:
New changes have not fixed the issue I showed above. Let me know how the email looks on mobile devices you have access to.
I suggest you use what was put previously:
padding: 12px 0 !important;
width: 220px !important;